src/utils/states.js

```diff
@@ -50,21 +50,26 @@ export const STATE_NAMES = [
   'All',
   'Success',
   'Failed',
+  'Finished',
   'Scheduled',
+  'ScheduledAlt',
   'Pending',
   'Running',
   'Cancelled',
   'Cancelling',
+  'Cached',
   'Retrying',
   'Queued',
   'Paused',
+  'InProgress',
   'Mapped',
   'Looped',
+  'Listening',
```

Comment:
I don't think we expose this state anywhere at the moment, so let's remove this for now